### ISSUE TITLE
Rule proposal: An assembly should have an AssemblyConfiguration attribute.

### ISSUE THREAD

**Issue by GregReddick** [DotNetAnalyzers/StyleCopAnalyzers#1642]
It should propose a warning if it is not present. The rule fix should add the following code to the AssemblyInfo.cs file if it is not present:
```
#if DEBUG
[assembly: AssemblyConfiguration("Debug")]
#else
[assembly: AssemblyConfiguration("Release")]
#endif
```

**Comment by sharwell** [comment]
Right now I'm :-1: for the same reason as #1640.

**Comment by vweijsters** [comment]
:-1: Similar to the discussion in #1640

**Comment by sharwell** [comment]
:arrow_right: See #1640 for my recommendation for proposing this rule as a code analysis warning rather than a StyleCop analyzer.
